client/src/components/Feedback/index.js

```diff
@@ -0,0 +1,145 @@
+  };
+
+  componentDidMount() {
+    axios
+      .get('/get-feedback')
+      .then(({ data }) => {
+        if (data.success) {
+          const { feedback } = data;
+          if (typeof feedback !== 'undefined') {
+            this.setState({ prevFeedback: feedback });
+            this.updateOptions(feedback);
+          }
+          this.setState({ loading: false });
+        }
+      })
+      .catch(err => console.log(err));
+  }
+
+  updateOptions = (feedbacks) => {
+    const { options } = this.state;
+    feedbacks.map((feedback) => {
+      options.map((option, i) => {
+        if (feedback === option.text) {
+          options[i].checked = true;
+        }
+      });
+    });
+    this.setState({ options });
+  };
+
+  handleCheckboxChange = (event) => {
+    const { options } = this.state;
+    options.map((option, i) => {
+      if (option.text === event.target.name) {
+        options[i] = { text: event.target.name, checked: event.target.checked };
+        this.setState({ options });
+      }
+    });
+  };
+
+  checkChange = (prev, current) => {
+    if (prev.length !== current.length) {
+      return true;
+    }
+    if (prev.length === 0 && current.length === 0) {
+      return false;
+    }
+
+    const sortedPrev = prev.sort();
+    const sortedCurrent = current.sort();
+    sortedPrev.map((prevObject, i) => {
+      if (prevObject !== sortedCurrent[i]) {
+        return true;
+      }
+    });
+    return false;
+  };
+
+  updateCurrentFeedback = () => {
+    const { options, currentFeedback } = this.state;
+    options.map((option) => {
+      if (option.checked) {
+        currentFeedback.push(option.text);
+      }
+    });
+    this.setState({ currentFeedback });
+  };
+
+  updateAirtaleFeedback = () => {
+    this.updateCurrentFeedback();
+    const { prevFeedback, currentFeedback } = this.state;
+    const changed = this.checkChange(prevFeedback, currentFeedback);
+    const { history } = this.props;
+
+    if (changed) {
+      axios
+        .put('/add-feedback', { feedback: currentFeedback })
+        .then(({ data }) => {
+          if (data.success) {
+            history.push('/item-list');
+          }
+        })
+        .catch(() => history.push('/error'));
+    }
+    history.push('/item-list');
+  };
+
+  render() {
+    const { options, loading } = this.state;
+
+    return (
+      <React.Fragment>
+        <Title />
+        <Header title="What do you want to do with your items?" />
+        {!loading && (
+          <CheckboxContainer>
+            {options.map((option, i) => (
+              <Checkbox
+                key={`${i}abc`}
+                name={option.text}
+                checked={option.checked}
+                onChange={this.handleCheckboxChange}
+              />
+            ))}
+          </CheckboxContainer>
+        )}
+        {loading && <Spinner />}
+        <BottomContainer>
+          <ThanksText>
+            Thanks for your commitment to a more sustainable world!
```

Comment:
This should only appear after they've selected their options.

Reply:
done